### PR TITLE
LFS: additions part 8

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: options/linux/include/scsi/sg.h [hunk 1]
```diff
@@ -42,6 +47,29 @@ struct sg_io_hdr {
     unsigned int info;
 };
 
+struct sg_scsi_id {
```
Comment (ikbenlike) [inline options/linux/include/scsi/sg.h:50]: is this sort of stuff not provided by the linux-headers option?

Comment (no92) [reply options/linux/include/scsi/sg.h:50]: While we don't have a `linux-headers` option any more, these header files are explicitly not provided by the kernel.

Comment (ikbenlike) [reply options/linux/include/scsi/sg.h:50]: in that case we would indeed have to provide these ourselves. I think that addresses all my concerns with this PR

**Review by ikbenlike — APPROVED** [review]
LGTM

File: options/glibc/include/bits/glibc/glibc_assert.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+#endif
+
+void __assert_fail_perror(int errno, const char *file, unsigned int line,
+		const char *function);
+
+#ifdef __cplusplus
+}
+#endif
+
+#endif // MLIBC_GLIBC_ASSERT_H
```
Comment (Dennisbonke) [inline options/glibc/include/bits/glibc/glibc_assert.h:15]: We should use C style comments here.

File: options/glibc/include/bits/glibc/glibc_assert.h [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+#ifdef NDEBUG
+
+#undef assert_perror
+#define assert_perror(ignore) ((void)0)
+
+#else // NDEBUG
```
Comment (Dennisbonke) [inline options/glibc/include/bits/glibc/glibc_assert.h:22]: We should use C style comments here.

File: options/glibc/include/bits/glibc/glibc_assert.h [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+#undef assert_perror
+#define assert_perror(errno) (!(errno) \
+		|| (__assert_fail_perror((errno), __FILE__, __LINE__, __func__), 0))
+
+#endif // NDEBUG
```
Comment (Dennisbonke) [inline options/glibc/include/bits/glibc/glibc_assert.h:28]: We should use C style comments here.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
LGTM outside 3 styling issues.

**Review by Geertiebear — APPROVED** [review]
LGTM

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**